pkg/proxy/util/utils.go

```diff
@@ -29,6 +26,9 @@ import (
 	"k8s.io/kubernetes/pkg/apis/core/v1/helper"
 	"k8s.io/kubernetes/pkg/features"
 	netutils "k8s.io/utils/net"
+	"net"
+	"strings"
+	"time"
```

Comment:
`net` and `strings` should stay where they were and `time` should go with them.

The general rule for import sorting in k8s is: all stdlib imports first (if any), then all non-stdlib-non-`k8s.io` imports (if any), then all `k8s.io` imports (if any), with a blank line between each group.